### DISCUSSION TITLE
Data loss or no tracking.

### DISCUSSION THREAD

**Discussion by orionhfast** [Security-Onion-Solutions/securityonion#13034] [General]
I don't know why it doesn't detect alerts if I've already set up the meerkat and everything related to the different IPs.
![Captura de pantalla 2024-05-17 102850](https://github.com/Security-Onion-Solutions/securityonion/assets/156486345/620c6b80-954b-4f40-b656-0e431731c3c9)

**Comment by dougburks** [comment]
Please try the troubleshooting steps in the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

If you need further help, please provide the results of each troubleshooting step and also more information about your system.
